esmvaltool/diag_scripts/examples/my_little_diagnostic.py

```diff
@@ -54,16 +101,20 @@ def plot_time_series(cfg):
     out_path = 'esmvaltool_users/valeriu/'   # edit as per need
     local_path = os.path.join(root_dir, out_path)
 
-    # get the files (simple case, one-variable only)
-    my_files_dict = _get_my_files(cfg)
+    # run your diagnostic - get the squared data
+    my_data_dict = _run_diagnostic(cfg)
```

Comment:
Instead of computing all data first and then plotting, consider computing the data for a plot and doing the plotting one at a time, this is much more memory efficient and therefore sets a better example.